src/content/roadmap/account-abstraction/index.md

```diff
@@ -84,12 +88,20 @@ The way wallets work would also change under EIP-4337. Instead of each wallet re
 
 <ExpandableCard title="EIP-2938: changing the Ethereum protocol to support account abstraction" eventCategory="/roadmap/account-abstract" eventName="clicked EIP-2938: changing the Ethereum protocol to support account abstraction">
 
-EIP-2938 aims to update the Ethereum protocol by introducing a new transaction type, `AA_TX_TYPE` that includes three fields: `nonce`, `target` and `data`, where `nonce` is a transaction counter, `target` is the entry point contract address and `data` is EVM bytecode. To execute these transactions, two new instructions (known as opcodes) have to be added to the EVM: `NONCE` and `PAYGAS`. The `NONCE` opcode tracks the transaction sequence and `PAYGAS` calculates and withdraws the gas required to execute the transaction from the contract's balance. These new features allow Ethereum to support smart contract wallets natively as the necessary infrastructure is built in to Ethereum's protocol.
+[EIP-2938](https://eips.ethereum.org/EIPS/eip-2938) aims to update the Ethereum protocol by introducing a new transaction type, `AA_TX_TYPE` that includes three fields: `nonce`, `target` and `data`, where `nonce` is a transaction counter, `target` is the entry point contract address and `data` is EVM bytecode. To execute these transactions, two new instructions (known as opcodes) have to be added to the EVM: `NONCE` and `PAYGAS`. The `NONCE` opcode tracks the transaction sequence and `PAYGAS` calculates and withdraws the gas required to execute the transaction from the contract's balance. These new features allow Ethereum to support smart contract wallets natively as the necessary infrastructure is built in to Ethereum's protocol.
 
 Note that EIP-2938 is currently not active. The community is currently favoring EIP-4337 because it does not require changes to the protocol.
 
 </ExpandableCard>
 
+<ExpandableCard title="EIP-3074: upgrading externally-owned accounts for account abstraction" eventCategory="/roadmap/account-abstract" eventName="clicked EIP-3074: upgrading externally-owned accounts for account abstraction">
+
+[EIP-3074](https://eips.ethereum.org/EIPS/eip-3074) aims to update Ethereum's externally-owned accounts by allowing them to delegate control to amsart contract. This means smart contract logic could approve transactions originating from an EOA. This would allow features such as gas-sponsoring and batched transactions. For this to work, two new opcoides have to be added to the EVM: `AUTH` and `AUTHCALL`. With EIP-3074 the benefits of a smart contract wallet are made available _without needing a contract_ - instead, a specific type of stateless, trustless, non-upgradeable contract known as an "invoker" handles the transactions.
```

Comment:
```suggestion
[EIP-3074](https://eips.ethereum.org/EIPS/eip-3074) aims to update Ethereum's externally-owned accounts by allowing them to delegate control to a smart contract. This means smart contract logic could approve transactions originating from an EOA. This would allow features such as gas-sponsoring and batched transactions. For this to work, two new opcodes have to be added to the EVM: `AUTH` and `AUTHCALL`. With EIP-3074 the benefits of a smart contract wallet are made available _without needing a contract_ - instead, a specific type of stateless, trustless, non-upgradeable contract known as an "invoker" handles the transactions.
```